.azure-pipelines/docker-sonic-mgmt.yml

```diff
@@ -0,0 +1,51 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+schedules:
+- cron: "0 8 * * *"
+  displayName: Build and push sonic-mgmt image
+  branches:
+    include:
+    - master
+  always: true
+
+trigger: none
```

Comment:
can you enable the trigger here? in this case, only when docker-sonic-mgmt folder changed, then trigger this.